### PR TITLE
Add OneAgent support for Dynatrace integration

### PR DESCRIPTION
**Pull request by ekremsekerci**
Updated the docs related to Dynatrace integration

### CODE REVIEW

File: content/en/docs/developerportal/operate/metrics/monitoring-with-apm/dynatrace-metrics.md [hunk 1]
```diff
@@ -43,18 +43,30 @@ To send your runtime information to Dynatrace, you need to set it up using envir
         The format for using the Dynatrace SaaS environment is similar to `https://<your-environment-id>.live.dynatrace.com`
 
         The format for using a Dynatrace managed environment is similar to  `https://<your-domain>/e/<your-environment-id>`
+   
+    2. DT_TENANT – *Required*
 
-    2. DT_PAAS_TOKEN – *Required*
+        The url of Dynatrace environment id, [Dynatrace Environment ID](https://www.dynatrace.com/support/help/get-started/monitoring-environment/environment-id)
```
Comment (pappachino) [inline content/en/docs/developerportal/operate/metrics/monitoring-with-apm/dynatrace-metrics.md]: `DT_TENANT` is the environment id from Dynatrace right and not the URL ? 🤔

Comment (ekremsekerci) [reply content/en/docs/developerportal/operate/metrics/monitoring-with-apm/dynatrace-metrics.md]: copy-paste mistake, fixed

**Review by pappachino — APPROVED** [review]

**Review by pappachino — APPROVED** [review]

**Comment by MarkvanMents** [comment]
Approved over Slack

**Comment by lucadesimoni** [comment]
Can i consider this working with Mendix Cloud and Dynatrace SaaS solution?

**Comment by ekremsekerci** [comment]
> Can i consider this working with Mendix Cloud and Dynatrace SaaS solution?

Hi @lucadesimoni, yes, we have multiple customers who use this integration with their Mendix apps. If you need more assistance, please feel free to file a ticket via https://support.mendix.com

**Comment by lucadesimoni** [comment]
Hi @ekremsekerci we have it now working for the metrics, but have some pending questions:
- we currently only get metrics and custom metrics
- but the most important would be logs, which are not passed to dynatrace, is that not supported? if not, how can we push for that on the roadmap?
very best 
luca

**Comment by feddovanede** [comment]
Hi @lucadesimoni! Your Customer Success Manager has reached out to us on this question as well. We have discussed the request and have put it on our backlog. We can't provide any timelines on this at this point. We will keep in contact with your CSM to keep you updated on the progress of this request.

**Pull request merged**